xml_inscriptions/transcribed/MD.Balt.WAG.L.23.17.xml

```diff
@@ -160,7 +160,7 @@
                         <ref type="external" target="https://art.thewalters.org/detail/16667/funeral-stele-with-latin-inscription-referring-to-mithra/">Walters Art Museum Web Entry</ref>
                     </bibl>
                     <bibl>
-                        <ref type="external" target="https://www.edr-edr.it/edr_programmi/res_complex_comune.php?do=book&amp;id_nr=EDR129704">EDR129704</ref>
```

Comment:
Why get rid of the https? Is it causing issues in the linking? (It shouldn't be, but if it is I'll take a look at the code that's handling it.)